### ISSUE TITLE
Start page specified for site not found: odraw:ROOT:index.adoc

### ISSUE THREAD

**Issue by rgleason** [opencpn-manuals/plugins#54]
ocpn_draw_pi   was specified in antora.yml  as  "name: odraw"
Because this leads to confusion about links, I changed it to  "name: ocpn_draw"
I have changed all the links in watchdog and weather_routing to work this way and will check ocpn_draw

Now this is happening.

check-sources  

==== Building plugins/sources/objsearch_pi/manual/site.yml
==== Building plugins/sources/ocpn_draw_pi/manual/site.yml
Start page specified for site not found: odraw:ROOT:index.adoc
==== Building plugins/sources/ocpndebugger_pi/manual/site.yml

**Comment by rgleason** [comment]
I've made a PR to Jon for site.yml  edit_url  to fix this.

https://github.com/jongough/ocpn_draw_pi/pulls

and it is accepted and closed.

**Comment by rgleason** [comment]
I have made changes to the appropriate nav.adoc index.adoc so the link works.  I am not going to push them however, yet.
The plugin seems to display properly  after antora site.yml and the links work fine.  
Links in watchdog and weather_routing work too.

However I am still getting this problem from check-sources.  Where should I loook  I have run out of ideas.
==== Building plugins/sources/ocpn_draw_pi/manual/site.yml
Start page specified for site not found: odraw:ROOT:index.adoc
.

**Comment by Rasbats** [comment]
In the repo site.yml you have: `start_page: odraw:ROOT:index.adoc` iso `start_page: ocpn_draw:ROOT:index.adoc` so you will find Jon and yourself will not be able to make a local build of just ocpn_draw.
For linking betas and main plugins the partials folder is used. Need to correct both `nav-plugins.adoc` and plugins.adoc. I will make those changes and test.

**Comment by Rasbats** [comment]
Also the edit_url should be `edit_url: https://github.com/jongough/ocpn_draw_pi/edit/master/{path}`. No longer using a fork in opencpn-manuals.

**Comment by Rasbats** [comment]
No ... we need to get site.yml in Jon's repo corrected before I can update anything.

**Comment by rgleason** [comment]
Before doing this post, I had made that change by making a PR to Jon and he accepted it!
See  10 hours ago  https://github.com/jongough/ocpn_draw_pi/commit/5903305fb8929332fbb150c354bdf7248ba453b7

`In the repo site.yml you have: start_page: odraw:ROOT:index.adoc iso start_page: ocpn_draw:ROOT:index.adoc so you will find Jon and yourself will not be able to make a local build of just ocpn_draw.
`
I have made this change already locally but had not pushed it!   
Running check-source  resulted in an error but that was because I had not pushed it.

What does check-source do?
Compares what to what?
I think that if you have not pushed this type of commit to opencpn-manual/plugins then there will be a problem wiith the checker, so I should disregard it in this case.
